### PR TITLE
[docs] Host authentication and UNIX socket address

### PR DESCRIPTION
**Pull request by ddhodge**
Add info on using local connections with host authentication and need to provide the path to the UNIX socket using -h flag

@netlify /preview/secure/authentication/host-based-authentication/

### CODE REVIEW

File: docs/content/preview/secure/authentication/host-based-authentication.md [hunk 1]
```diff
@@ -97,6 +97,29 @@ hostnossl  database  user  IP-address  netmask  auth-method  [auth-options]
...
+2023-09-05 13:56:20.154 UTC [1261] LOG:  listening on Unix socket "/tmp/.yb.127.0.0.1:5433/.s.PGSQL.5433"
+```
+
+If you use ysqlsh, use the `-h` flag, and pass in the first part of the path (in the preceding example, `/tmp/.yb.127.0.0.1:5433/`) as follows:
+
+```sh
+./bin/ysqlsh -h /tmp/.yb.127.0.0.1:5433
+```
+
+For psql you also need specify the port number:
```
Comment (aishwarya24) [inline docs/content/preview/secure/authentication/host-based-authentication.md:117]:
```suggestion
For psql, you also need specify the port number as follows:
```

**Review by aishwarya24 — APPROVED** [review]
LGTM. Back port to stable and older versions as required.

File: docs/content/preview/secure/authentication/host-based-authentication.md [hunk 2]
```diff
@@ -97,6 +97,29 @@ hostnossl  database  user  IP-address  netmask  auth-method  [auth-options]
```
Comment (jasonyb) [inline docs/content/preview/secure/authentication/host-based-authentication.md:97]: out-of-scope: local, host, hostssl, hostnossl probably belong in a subcategory similar to how auth methods are

File: docs/content/preview/secure/authentication/host-based-authentication.md [hunk 3]
```diff
@@ -97,6 +97,29 @@ hostnossl  database  user  IP-address  netmask  auth-method  [auth-options]
 
 This record matches connection attempts made via the UNIX socket.
 
+Similarly to PostgreSQL, YugabyteDB supports opening connections via the UNIX sockets when local authentication is used. However, unlike PostgreSQL, YugabyteDB requires ysqlsh, psql and other tools to provide the full path to the socket location when using the following authentication methods:
+
+- local authentication using [peer](#peer)
+- local authentication using [trust](#trust)
+
+The path to the socket location is written to YB-TServer logs, similar to the following:
```
Comment (jasonyb) [inline docs/content/preview/secure/authentication/host-based-authentication.md:105]: This is one of many ways to get the socket path

**Review by dmagda — APPROVED** [review]
Looks good to me, thanks!

**Review by jasonyb — APPROVED** [review]

**Pull request merged**